lombok.config

```diff
@@ -0,0 +1,4 @@
+# stop lombok from looking at parent directories for more configuration files
```

Comment:
As discussed previously on slack (and on some other Pull Requests), you should not use lombok as we would like the focus to be on the code that you write and not on code generation.